.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,223 @@
+name:        "Bug Report"
+description: "Create a new Bug report. Have you found an unexpected behavior?, use this form."
```

```suggestion
description: "Create a new bug report. Have you found an unexpected behavior?, use this form."
```

I don't think we need random capital words